Makefile

```diff
@@ -127,13 +111,38 @@ $(LOCALBIN):
 	mkdir -p $(LOCALBIN)
 
 ## Tool Binaries
+KIND ?= $(LOCALBIN)/kind
+KUBECTL ?= $(LOCALBIN)/kubectl
 KUSTOMIZE ?= $(LOCALBIN)/kustomize
 CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 ENVTEST ?= $(LOCALBIN)/setup-envtest
+KUBECONFIG_PATH ?= $(LOCALBIN)/kubeconfig
+export KUBECONFIG=$(KUBECONFIG_PATH)
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v5.0.0
 CONTROLLER_TOOLS_VERSION ?= v0.11.3
+KUBECTL_VERSION ?= v1.27.0
+KIND_VERSION ?= v0.19.0
+KIND_CLUSTER_NAME ?= frr-k8s
+
+.PHONY: install
+install: kubectl manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
```

Comment:
nit: remove the `~/.kube/config` mentions now that we have a different kubeconfig

Reply:
you can change it to `bin/kubeconfig` instead of removing it completely